robottelo/config/virtwho.py

```diff
@@ -0,0 +1,368 @@
+"""Define and instantiate the configuration class for virtwho hypervisors."""
+import logging.config
+import os
+
+from robottelo.config.base import FeatureSettings
+from robottelo.config.base import get_project_root
+from robottelo.config.base import ImproperlyConfigured
+from robottelo.config.base import INIReader
```

Comment:
Just for your information, there is plan to move to Dynaconf, which will store configuration in YAML files. See #8003 . At some unspecified time in the future, `robottelo.config.base` will be removed.

I don't want to block this PR, but I want to give you heads up and opportunity to re-think how these configurations will be stored. YAML gives you opportunity to store values in multilevel namespaces, make values proper complex data structures (like lists or dictionaries) or re-use part of configuration in many places.

Reply:
Yes, I saw your PR when it was raised cuz I also focus on the Dynaconf integration job, and really happy with your fantastic progress.

We did think about how to set our configs, and the job move to Dynaconf is still in progress,  so just as I said in the description of this PR, we will also update our configurations once the Dynaconf PR is merged/the `robottelo.config.base` is removed. We will sync up with the SATQE team, do you think this will be ok? Anyway, thanks for bringing up this.

Reply:
Yes, syncing with dynaconf config later on is OK. Dynaconf PR is not blocking this one, but I wanted to ensure you are aware of it and that you plan for it accordingly.

Reply:
Gotcha, thanks